### PR TITLE
Add Protocol Definitions for Management Center Operations

### PR DESCRIPTION
**Pull request by emre-aydin**
Also merge `metrics.yaml` into `MC.yaml` which contains all MC related client operations.

### CODE REVIEW

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+id: 40
+name: MC
+methods:
+  - id: 1
+    name: ChangeClusterStateRequest
```
Comment (asimarslan) [inline protocol-definitions/MC.yaml]: please use java method naming convention here.

File: protocol-definitions/MC.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    since: 2.0
+    doc: |
+      Changes the state of a cluster.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: -1
+      params:
+        - name: newState
+          type: Data
```
Comment (asimarslan) [inline protocol-definitions/MC.yaml]: custom type

File: protocol-definitions/MC.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        - name: mapName
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Name of the map.
+    response:
+      params:
+        - name: response
+          type: Data
```
Comment (asimarslan) [inline protocol-definitions/MC.yaml]: custom type

File: protocol-definitions/MC.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      partitionIdentifier: -1
+      params:
+        - name: mapName
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Name of the map.
+        - name: newMapConfig
+          type: Data
```
Comment (asimarslan) [inline protocol-definitions/MC.yaml]: custom type

File: protocol-definitions/MC.yaml (hunk 2 above)
Comment (emre-aydin) [reply protocol-definitions/MC.yaml]: used `String` as discussed with @mdumandag

**Comment by emre-aydin** [comment]
Waiting for https://github.com/hazelcast/hazelcast/pull/15650 to be merged for unifying `MC.yaml ` and `Metrics.yaml`

**Comment by blazember** [comment]
@emre-aydin https://github.com/hazelcast/hazelcast/pull/15650 has been merged.

**Comment by emre-aydin** [comment]
@mdumandag @asimarslan ready for another review.

**Comment by emre-aydin** [comment]
@asimarslan I can't seem to merge this one because of lack of privileges I guess. I've merged the core PR (https://github.com/hazelcast/hazelcast/pull/15662), can you please merge this one as well?

**Pull request merged**